static/app/views/starfish/views/webServiceView/endpointOverview/issuesTable.tsx/index.tsx

```diff
@@ -0,0 +1,39 @@
+import GroupList from 'sentry/components/issues/groupList';
+import {IssueCategory} from 'sentry/types';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {getDateFilters} from 'sentry/views/starfish/utils/dates';
+
+type Props = {
+  issueCategory?: IssueCategory;
+  transactionName?: string;
+};
+
+function IssuesTable(props: Props) {
+  const {transactionName, issueCategory} = props;
+  const organization = useOrganization();
+  const pageFilters = usePageFilters();
+  const {startTime, endTime} = getDateFilters(pageFilters);
+
+  const queryConditions: string[] = [
+    'is:unresolved',
+    ...(issueCategory ? [`issue.category:${issueCategory}`] : ['']),
+    ...(transactionName ? [`transaction:${transactionName}`] : ['']),
+    `start:${startTime.format('YYYY-MM-DDTHH:mm:ss')}`,
+    `end:${endTime.format('YYYY-MM-DDTHH:mm:ss')}`,
+  ];
+
+  return (
+    <GroupList
+      orgId={organization.slug}
```

Comment:
can we limit to 5 rows

Reply:
👍 Yeah we can, and i can remove pagination as well if its always at most 5.